.github/analytics/issue_activity_since_date.gql

```diff
@@ -0,0 +1,56 @@
+{
+  repository(owner: "_REPO_OWNER_", name: "_REPO_NAME_") {
```

Comment:
Could you add some comments to these files explaining what exactly they are querying?